setup.py

```diff
@@ -496,6 +496,12 @@ def write_version(filename: str = str(AIRFLOW_SOURCES_ROOT / "airflow" / "git_ve
     "aiobotocore>=2.1.1",
 ]
 
+s3fs = [
```

Comment:
I agree, but I thought I was doing that :-).

Reply:
Should be done now

Reply:
Actually @bolkedebruin , I change  my mind about about this after looking back at the setup.py, I think it is needed, otherwise we will not have it in the image and it should be added to `devel` similarly as `aiobotocore`

Reply:
Arghh :-). There was a reason why I was struggling with it. Okay I'll re-add it